docs/cli.rst

```diff
@@ -0,0 +1,100 @@
+.. _cli:
+
+Command Line Interface
+======================
+
+Tealish has a CLI::
+
+    % tealish
+    Usage: tealish [OPTIONS] COMMAND [ARGS]...
+
+    Tealish Compiler & Tools
+
+    Options:
+    -q, --quiet  Only print warnings and errors
+    -h, --help   Show this message and exit.
+
+    Commands:
+    build     Compile .tl to .teal & assemble .teal to .tok (bytecode) &...
+    compile   Compile .tl to .teal
+    format    Rewrite .tl file using standard tealish style
+    html      Output HTML of Tealish & Teal source
+    langspec  Tools to support new Teal versions by updating the langspec file
+
+
+Compiling
+---------
+
+Basic compiling .tl to .teal::
+
+    tealish compile examples/counter_prize.tl
+
+This outputs examples/build/counter_prize.teal
```

Comment:
FYI: This path is updated.
Should we update it? I think, there is no need.

Reply:
Yeah it's just an example command, it's fine.